### PR TITLE
fix: fixed title cut off on larger resolutions

### PR DESCRIPTION
**Pull request by pi0trpietruszewski**

### CODE REVIEW

**Review by IdaszakDaniel — APPROVED** [review]

File: examples/expo/src/constants/screenStyles.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const screenStyles = StyleSheet.create({
   message: {
     color: colors.white,
     fontSize: constants.responsiveWidth(11),
-    lineHeight: 48,
+    lineHeight: constants.responsiveWidth(11) * 1.2,
```
Comment (marcinkornek) [inline examples/expo/src/constants/screenStyles.js:62]: why this value? And why fontSize depends on width?

Comment (Patys) [reply examples/expo/src/constants/screenStyles.js:62]: or we can use system font scale? https://reactnative.dev/docs/pixelratio 🤔

Comment (pi0trpietruszewski) [reply examples/expo/src/constants/screenStyles.js:62]: i just used current solution

Comment (pi0trpietruszewski) [reply examples/expo/src/constants/screenStyles.js]: fixed

**Review by Patys — APPROVED** [review]

**Review by marcinkornek — APPROVED** [review]

**Pull request merged**